conkit/plot/contactplotplus.py

```diff
@@ -0,0 +1,267 @@
+from bokeh.plotting import figure, show, output_file
+from bokeh.models import Range1d, Legend, ColumnDataSource
+from bokeh.models.tools import HoverTool
+
+
+class ContactPlotPlusFigure(object):
+
+    def __init__(self, seq, conpred, factor=1):
+
+        self.conpred = conpred
+        self.conservpred = None
+        self.mempred_i = None
+        self.mempred_m = None
+        self.mempred_o = None
+        self.sspred_h = None
+        self.sspred_c = None
+        self.sspred_e = None
+        self.dispred_o = None
+        self.dispred_dis = None
+        self.conservpred = None
+        self.reference = None
+        self.contact_w_reference_mis = None
+        self.contact_w_reference_match = None
+        self.reference_points = None
+        self.contact_circles = None
```

Comment:
why? there are duplicates, this is not needed if it's not provided via the constructor